### ISSUE TITLE
Locked Column Width is changed when Column Virtualization is on

### ISSUE THREAD

**Issue by thangpham-kms** [telerik/kendo-react#312]
## I'm submitting a...

* Bug report 

## Current behavior
The locked column width has broken when scrolling horizontally. 

## Expected behavior

The locked column width should concise during scrolling.

## Minimal reproduction of the problem with instructions
[https://stackblitz.com/edit/react-li2omr?embed=1&file=app/main.jsx](https://stackblitz.com/edit/react-li2omr?embed=1&file=app/main.jsx)

Scroll until it reaches over field 7.  Notice the width of the second locked column has been changed. 

## What is the motivation or use case for changing the behavior?
When column virtualization is on and there are locked columns, the column width should be rendered accordingly with the visible columns.

## Environment

Browser:
- Chrome (desktop) version 76

System:
- Platform:  Mac

**Comment by thangpham-kms** [comment]
When Column Virtualization is on, it only renders visible column on the view, while the `<colgroup>` is still remaining unchanged.

```html
<table>
    <!-- the colgroup is not updated  -->
    <colgroup>
        <col width="200px"> <!-- locked column 3 width is 200px instead of 50px -->
        <col width="150px"> <!-- locked column 4 width is 150px instead of 100px -->
        <col width="50px">
        <col width="100px">
        <col width="150px">
        <col width="150px">
        <col width="150px">
        <col width="150px">
        <col width="150px">
        <col width="150px">
    </colgroup>
    <tbody>
    <!-- only render visible td  -->
    <tr class="k-master-row">
        <td colspan="1">R1:C3</td>
        <td colspan="1">R1:C4</td>
        <td colspan="4">R1:C6</td>
        <td colspan="1">R1:C7</td>
        <td colspan="1">R1:C8</td>
        <td colspan="1">R1:C9</td>
        <td colspan="1">R1:C10</td>
    </tr>
    </tbody>
</table>

```

**Comment by nstoychev** [comment]
@thangpham-kms

Fixed in version [3.9.0](https://k2build.openstack.progress.com/kendo-react-ui/components/changelogs/ui-for-react/#toc-390).

Here is the updated example - https://stackblitz.com/edit/react-li2omr-hlrzf3?file=app%2Fmain.jsx
